### ISSUE TITLE
[Component] Button - toggle feature

### ISSUE THREAD

**Issue by thomas-leguellec** [adevinta/spark-android#945]

**Comment by thomas-leguellec** [comment]
Need a fix combination.

**Comment by thomas-leguellec** [comment]
@turolopezsanabria @Bibisirius @ChrisDesignOps @jcatdom 
Even if it's noted as "Done" for the specs, it's not the case for the button.
Can you update the specs please and we will work on this ticket on Android side later.
Thanks!


![Image](https://github.com/adevinta/spark-android/assets/123980564/8dff9929-97f4-4801-88b5-fb4d50d538e0)

**Comment by thomas-leguellec** [comment]
@JeremiasUX @Bibisirius @jcatdom @ChrisDesignOps Can you tell us if it's ready or not please? Thanks

**Comment by JeremiasUX** [comment]
Yes, it is ready - https://www.figma.com/design/0QchRdipAVuvVoDfTjLrgQ/Component-Specs-of-Spark?node-id=2340-22746&t=1z2MC4BwrQbPCRE5-4

**Comment by soulcramer** [comment]
@JeremiasUX What we miss to be able to handle this specs is a lear definition like for the chip & icon toggle button on what style can be selected (how does a filled or contrast button renders in the selected state?)

**Comment by thomas-leguellec** [comment]
No needed anymore. Let's remove this one and see in the future if consumer has a specific request.

Let's just provide an example on Catalog App.
